.gitignore

```diff
@@ -5,3 +5,4 @@ munge.key
 ssh_host_*key*
 **/.DS_Store
 /kraken-build
+**/*.swp
```

Comment:
While this isn't a bad idea, it should probably be its own PR.

Reply:
See #156